src/meta/src/barrier/mod.rs

```diff
@@ -214,7 +214,7 @@ type CheckpointPost<S> = (
 struct UncommittedMessages<S: MetaStore> {
     uncommitted_checkpoint_post: VecDeque<CheckpointPost<S>>,
     /// Ssts that need to commit with next checkpoint.
-    uncommitted_ssts: Vec<LocalSstableInfo>,
+    uncommitted_ssts: VecDeque<LocalSstableInfo>,
```

Comment:
Please also add documentation here about what order we preserve and why the ordering matters.

Reply:
+1